### PR TITLE
[#169167640] Add CIE authorization complete confirmation screen

### PR DESCRIPTION
**Pull request by matgentili**
This pr introduces a confirmation screen after cie authorization complete.

valid cie

### CODE REVIEW

File: locales/en/index.yml [hunk 1]
```diff
@@ -213,6 +213,8 @@ authentication:
     card5-content: In this experimental version you can find in the upper-right corner
       the icons to report bugs or start a chat with the development team.
       We're waiting for your feedbacks!
+    ok: Ok
+    cardOk: Ok, your card is valid! Wait a few more seconds ...
```
Comment (Undermaken) [inline locales/en/index.yml]:
```suggestion
    cardOk: Ok, your card is valid! Wait a few more seconds..
```
https://projects.invisionapp.com/share/FZUE5VC8JNQ#/screens

Comment (Undermaken) [inline locales/en/index.yml]:
```suggestion
    cieCardValid: Ok, your card is valid! Wait a few more seconds ...
```

File: ts/screens/authentication/CieConfirmScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+          <H1>{I18n.t("authentication.landing.ok")}</H1>
+          <Text style={styles.text}>
+            {I18n.t("authentication.landing.cardOk")}
+          </Text>
+        </View>
+        <View spacer={true} extralarge={true} />
+        <View style={styles.containerBox}>
+          <View style={styles.container}>
+            <Image
+              source={require("../../../img/landing/place-card-illustration.png")}
```
Comment (Undermaken) [inline ts/screens/authentication/CieConfirmScreen.tsx]: try to put it out of the render method and create a const instead

`const cieImage = require("../../../img/landing/place-card-illustration.png")`

File: ts/screens/authentication/CieConfirmScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+              size={50}
+            />
+          </View>
+        </View>
+      </BaseScreenComponent>
+    );
+  }
+}
+
+export default connect()(CieConfirmScreen);
```
Comment (Undermaken) [inline ts/screens/authentication/CieConfirmScreen.tsx]:
```suggestion
export default CieConfirmScreen;
```
This screen has no redux dependencies, so you can avoid connect

File: ts/screens/wallet/WalletHomeScreen.tsx [hunk 4]
```diff
@@ -315,9 +316,10 @@ class WalletHomeScreen extends React.Component<Props, never> {
       <Button
         block={true}
         onPress={
-          pot.isSome(potWallets)
-            ? this.props.navigateToPaymentScanQrCode
-            : undefined
+          this.props.navigateToCIEOK
```
Comment (Undermaken) [inline ts/screens/wallet/WalletHomeScreen.tsx]: you can revert all these changes.
For test purpose I will build an ad-hoc version that allows to display this screen

File: ts/store/actions/navigation.ts [hunk 5]
```diff
@@ -233,3 +233,8 @@ export const navigateToPaymentManualDataInsertion = (
     routeName: ROUTES.PAYMENT_MANUAL_DATA_INSERTION,
     params
   });
+
+export const navigateToCieOK = () =>
```
Comment (Undermaken) [inline ts/store/actions/navigation.ts]:
```suggestion
export const navigateToCieValid = () =>
```

File: ts/screens/wallet/WalletHomeScreen.tsx (hunk 4 above)
Comment (matgentili) [reply ts/screens/wallet/WalletHomeScreen.tsx]: Yes,  it was just the fastest way I had already available

Comment (Undermaken) [reply ts/screens/wallet/WalletHomeScreen.tsx]: Yes I know but this PR will be merged and when it is merged that version is available to all beta tester. If we miss something we could publish a wrong version of the app

File: ts/navigation/routes.ts [hunk 6]
```diff
@@ -42,6 +42,7 @@ const ROUTES = {
   PAYMENT_PICK_PAYMENT_METHOD: "PAYMENT_PICK_PAYMENT_METHOD",
   PAYMENT_CONFIRM_PAYMENT_METHOD: "PAYMENT_CONFIRM_PAYMENT_METHOD",
   PAYMENT_PICK_PSP: "PAYMENT_PICK_PSP",
+  CIE_CONFIRM_SCREEN: "CIE_CONFIRM_SCREEN",
```
Comment (Undermaken) [inline ts/navigation/routes.ts]:
```suggestion
  CIE_VALID_SCREEN: "CIE_VALID_SCREEN",
```

**Pull request merged**